packages/react/src/components/MapCard/storyFiles/MapboxExample.jsx

```diff
@@ -0,0 +1,315 @@
+import React, { useRef, useEffect, useState } from 'react';
+import {
+  Events32,
+  Cloud32,
+  Cloudy32,
+  Fog32,
+  Hail32,
+  PartlyCloudy32,
+  Rain32,
+  Sun32,
+} from '@carbon/icons-react';
+import { Accordion, AccordionItem } from 'carbon-components-react';
+// The MapboxExample is not exported and is only used by StoryBook
+/* eslint-disable-next-line import/no-extraneous-dependencies */
+import mapboxgl from 'mapbox-gl';
+import PropTypes from 'prop-types';
+
+import MapCard from '../MapCard';
+
+import Optionsfield from './Optionsfield';
+import './mapbox-example.scss';
+
+mapboxgl.accessToken =
+  'pk.eyJ1IjoiZGF2aWRpY3VzIiwiYSI6ImNrbTN4OWpsZTBjYm0ybnBsaWZkemV6MmgifQ.jpqC4rJzYG6CY3IXc9NLuw';
```

Comment:
is hard-coding the access token here, okay?

Reply:
I don't know, I haven't touched this part actually. @davidicus what do you think?

Reply:
This is the access token from their docs. Should be fine.